lib/validation.js

```diff
@@ -59,13 +59,15 @@ module.exports = function validation (yargs, usage, y18n) {
   self.missingArgumentValue = function missingArgumentValue (argv) {
     const defaultValues = [true, false, '', undefined]
     const options = yargs.getOptions()
-
     if (options.requiresArg.length > 0) {
       const missingRequiredArgs = []
 
       options.requiresArg.forEach((key) => {
-        const value = argv[key]
+        // if the argument is not set in argv no need to check
+        // whether a right-hand-side has been provided.
+        if (Object.keys(argv).indexOf(key) === -1) return
```

Comment:
Why not `argv.hasOwnProperty(key)`?

Reply:
@jddxf no good reason, will update the code accordingly.